core/domain/image_services.py

```diff
@@ -0,0 +1,103 @@
+# coding: utf-8
+#
+# Copyright 2014 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Service for managing operations on images."""
+
+from core.domain import fs_domain
+from core.platform import models
+import feconf
+
+gae_image_services = models.Registry.import_gae_image_services()
```

Comment:
This is not how one imports platform-layer services. Follow existing codebase conventions please.

Reply:
The other imports are in this manner only. [See](https://github.com/oppia/oppia/blob/develop/core/domain/feedback_services.py#L35)

Reply:
Ah, sorry, I know what threw me off. It should be image_services rather than gae_image_services, and import_image_services() rather than import_gae_image_services().

Reply:
We already have an image_service in core/domain so we can't keep image_services for this one.
Out of both of them whose name shall I change ?

Reply:
Let's just move this stuff to exp_services. I'm not really sure that we need an image_services in the domain layer.